third_party/move/move-compiler-v2/tests/checking/typing/vector_mismatched_args.exp

```diff
@@ -0,0 +1,31 @@
+
+Diagnostics:
+error: `integer` not compatible with `bool` in vector expression
+  ┌─ tests/checking/typing/vector_mismatched_args.move:7:19
+  │
+7 │         vector[0, false];
+  │                   ^^^^^
+
+error: expected `u64` but found `u8` in vector expression
+  ┌─ tests/checking/typing/vector_mismatched_args.move:8:21
+  │
+8 │         vector[0u8, 0u64, 0u128];
```

Comment:
This one is very confusing, and is more urgent.

Reply:
This must be outdated as well, because in my source, the direction is correct. BTW, instead of saying "this is confusing" you could just identify the bug and say that the types are obviously in the wrong order, and it should be `expected u8 but found u6`. Confusing in this context sounds like this would be intended. Indeed it is not.

Reply:
Sorry, my knowledge of Move typing is incomplete, so I didn't immediate realize that it was so simple as being the right types but in the wrong order.